src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();

```

Comment:
```suggestion
    private final Random random = new Random();
```

Reply:
I don't understand, why i should change it to non-static if code works good. what problems it can cause?

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/6c33a7e9-6aa7-4da6-94b2-78873b77e2f4)